### DISCUSSION TITLE
DAI 2023-2024 - My IntelliJ + Maven project - Ahmad Jano

### DISCUSSION THREAD

**Discussion by ahmadjano** [heig-vd-dai-course/heig-vd-dai-course#58] [Show and tell]
[My IntelliJ + Maven project](https://github.com/ahmadjano/java-intellij-idea-and-maven)

**Comment by ludelafo** [comment]
I guess your project is private as I cannot see the content. If you want me to have a look at it, please invite me to your project. :)

**Reply by ahmadjano** [reply]
Oops didn't notice. It's public now.

**Reply by ludelafo** [reply]
Thank you!

The only thing I would add is the commands to download the dependencies and build the application through the command line with Maven. This will allow everyone to build and use your application, even those who don't want/have IntelliJ IDEA to execute the Buid/Debug Configuration.
